layouts/brukeropplevelse-og-datadeling/list.html

```diff
@@ -0,0 +1,144 @@
+{{ define "main" }}
+
+<div>
+  {{ with .Site.GetPage "/brukeropplevelse-og-datadeling" }}
```

Comment:
The whole list is hardcoded and not based on structure and metatadata. Is that ok?

Reply:
Oh, sorry, this was actually a single page type, not a list. the file name is now changed to reflect the type.

Reply:
if that was what you meant? 🤔

Reply:
Yup, but that's for a later PR